gnomad/assessment/validate_vep.py

```diff
@@ -0,0 +1,179 @@
+"""script to validate VEP annotation on protein-coding genes."""
+import argparse
+import logging
+from datetime import datetime
+
+import hail as hl
+
+from gnomad.resources.grch38.reference_data import ensembl_interval
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def count_variant_per_interval(
```

Comment:
I would modify this to be more general since it's in `gnomad_methods`, so maybe have it only take in a `vep_ht` and an `interval_ht`. Remove all checks for release and version since it's just taking in the `vep_ht`. Move to https://github.com/broadinstitute/gnomad_methods/blob/main/gnomad/assessment/validity_checks.py